packages/api/src/Database/seeders/entities.ts

```diff
@@ -2,11 +2,17 @@ import faker from 'faker';
 
 import { UserService } from '../../User/User.service';
 
-export const seedUsers = async (userService: UserService) => userService.create({
-  firstName: faker.name.firstName(),
-  lastName: faker.name.lastName(),
-  email: 'test@test.com',
-  password: 'test123'
-});
+export const seedUsers = async (userService: UserService) => Promise.all(Array(5).fill(undefined).map(async (_, i) => {
```

Comment:
Can we make this `5` a parameter? Might want to scale it. It can default to 5, but should be configurable